addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -380,12 +380,14 @@ def _get_component_data(self, parent_bom, parent_product, warehouse, bom_line, l
         }
 
     @api.model
-    def _get_quantities_info(self, product, bom_uom, product_info, parent_bom=False, parent_product=False):
-        return {
+    def _get_quantities_info(self, product, bom_uom, product_info, parent_bom=False, parent_product=False, additional_product_metadata={}):
```

Comment:
You can't change a method signature in stable because we don't know if the method was overridden by custom module.
Also [it's not a good practice to use a dict as default parameter's value in python](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) and changing the value of a dict silently during the call of a method is also something to avoid